framework/src/modules/chain/dpos/delegates_list.js

```diff
@@ -88,11 +88,30 @@ class DelegatesList extends EventEmitter {
 	}
 
 	async deleteDelegateListUntilRound(round) {
+		this.deleteDelegateListsFromCache(r => r < round);
 		await this.storage.entities.RoundDelegates.delete({
 			round_lt: round,
 		});
 	}
 
+	async deleteDelegateListAfterRound(round, tx) {
+		this.deleteDelegateListsFromCache(r => r > round);
+		await this.storage.entities.RoundDelegates.delete(
+			{
+				round_gt: round,
+			},
+			tx,
+		);
+	}
+
+	deleteDelegateListsFromCache(filterCb) {
+		Object.keys(this.delegateListCache)
+			.filter(filterCb)
```

Comment:
Is this a filter function? Mabye `filterFn`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#Syntax in the api description they called it callback so did i :D

Reply:
I hate any reference to the callback era :D I saw some other implementations where they call it `lambdaFn` `fn` `filterFn`, feel free to pick what feels appropriate or use the standard by Mozilla :)